src/lib.rs

```diff
@@ -1,8 +1,9 @@
 //! # Darling
-//! Darling is a tool for declarative attribute parsing in proc macro implementations.
 //!
+//! Darling is a tool for declarative attribute parsing in proc macro implementations.
```

Comment:
Are these whitespace changes necessary?

Reply:
No, but they do help make the doc comment more readable in an editor. I can remove them if you'd prefer.